test/cluster_monitor.go

```
@@ -119,6 +119,31 @@ type ConditionsSummary struct {
+	cmd := exec.Command("kubectl", "--context", kubeContext, "get", "ns", "--request-timeout=5s")
+	output, err := cmd.CombinedOutput()
+	if err == nil {
+		return nil
+	}
+
+	combined := string(output) + " " + err.Error()
+	netErr := DetectNetworkError(combined)
+	if netErr != nil {
+		return fmt.Errorf("management cluster unreachable (%s): %s", netErr.ErrorType, netErr.Message)
+	}
+	return fmt.Errorf("management cluster health check failed: %v", err)
```

golangci-lint (2.12.2)

[error] non-wrapping format verb for fmt.Errorf. Use `%w` to format errors

(errorlint)